scripts/uninstall.sh

```diff
@@ -255,7 +311,7 @@ askreboot(){
 	if [[ ${REPLY} =~ ^[Yy]$ ]]; then
 		printf "\\nRebooting system...\\n"
 		sleep 3
-		shutdown -r now
+		reboot
```

Comment:
Even tho nowadays reboot is widely accepted in any distro release and unless the standard has changed, `shutdown -r now` should be the correct way.